### PR TITLE
Update recipe for py312_patch

### PR DESCRIPTION
**Pull request by cbouss**
Preparation for py312 release.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -40,12 +40,12 @@ test:
     - ansi2html
   commands:
     - pip check
-    - nosetests tests/test_ansi2html.py
+    - nosetests tests/test_ansi2html.py  # [py<312]
```
Comment (ryanskeith) [inline recipe/meta.yaml]: This is interesting.  It looks like upstream may not use nose (tox really) and if they don't we could probably just use regular pytest.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -40,12 +40,12 @@ test:
...
     - ansi2html --help
     - ansi2html --version | grep -q {{ version }}
   requires:
     - mock >=0.8
-    - nose
+    - nose  # [py<312]
```
Comment (ryanskeith) [inline recipe/meta.yaml]: Again, we could go with pytest and pytest-cov as specific [here](https://github.com/pycontribs/ansi2html/blob/b4224cd6a36e99117d09ad365d5a180ebf29e356/setup.cfg#L51C2-L51C2) and [here](https://github.com/pycontribs/ansi2html/blob/b4224cd6a36e99117d09ad365d5a180ebf29e356/tox.ini#L19).

**Review by ryanskeith — COMMENTED** [review]
I suggest removing nose.

**Comment by cbouss** [comment]
> I suggest removing nose.

done, switched to pytest

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Review by Jrice1317 — APPROVED** [review]
LGTM

**Pull request merged**